sections/installation.js

```diff
@@ -390,4 +390,36 @@ export default [
       });
     `
   },
+  {
+    type: "heading",
+    size: "md",
+    content: "log",
+    href: "Installation-log"
+  },
+  {
+    type: "text",
+    content: [
+      "Knex contains some internal log functions for printing warnings, errors, deprecations, and debug information when applicable.",
+      "These log functions typically log to the console, but can be overwritten using the log option and providing alternative functions.",
```

Comment:
what will happen if you pass log parameter with only a subset of functions, e. g. without "warn"? Does default logging work in such case or will it break?